proto/internal/temporal/server/api/persistence/v1/task_queues.proto

```diff
@@ -116,6 +116,9 @@ message DeploymentData {
 // Container for all persistent user data that varies per task queue type within a family.
 message TaskQueueTypeUserData {
     DeploymentData deployment_data = 1;
+
+    // Perist TaskQueueConfig that comes as part of the UpdateTaskQueueConfigApi.
```

Comment:
typos:
```suggestion
    // Persist TaskQueueConfig that comes as part of the UpdateTaskQueueConfig api.
```

but I don't think this comment really adds anything, the comment on the message says it's for all persisted user data, and it's obvious a TaskQueueConfig